src/sentry/static/sentry/app/views/eventsV2/data.tsx

```diff
@@ -72,7 +72,7 @@ export const TRANSACTION_VIEWS: Readonly<Array<EventViewv1>> = [
       sort: ['-count'],
       query: 'event.type:transaction',
     },
-    tags: ['release', 'project.name', 'user.email', 'user.ip'],
+    tags: ['release', 'project.name', 'user.email', 'user.ip', 'environment'],
```

Comment:
Having the release,environment at the end or the beginning consistently would be :+1: